src/openforms/submissions/models/submission.py

```diff
@@ -14,6 +15,7 @@
 
 from openforms.config.models import GlobalConfiguration
 from openforms.formio.datastructures import FormioConfigurationWrapper
+from openforms.formio.utils import iterate_data_with_components
```

Comment:
please expose this function via `openforms.formio.service` which is the place for public API. It's okay to just re-export it from the utils module, but we should avoid importing from other app modules other than the `service` module.